src/routes/formsgSiteCreation.ts

```diff
@@ -157,9 +167,83 @@ export class FormsgRouter {
     await mailer.sendMail(email, subject, html)
   }
 
+  cloneSiteToEfs: RequestHandler<
```

Comment:
Could we shift this to a `formsgSiteClone.ts`? We keep one file -> path

Reply:
surev